### DISCUSSION TITLE
typescript버전에 따른 에러 검출 문제

### DISCUSSION THREAD

**Discussion by koldshine** [naverz/zepeto-studio-kor#1035] [5. Q&A]
vscode로 개발중이고 typescript는 공식팁 내용처럼 4.3.5를 선택했습니다

아래 이미지처럼 typescript 4.3.5를 선택하면 코드를 잘 못 구현해도 에러검출이 되지 않습니다
![화면 캡처 2022-07-19 142030](https://user-images.githubusercontent.com/105689042/179682752-1301c060-38fc-4102-bf29-747f31c3db06.png)

4.7.3을 선택하면 에러검출이 잘 됩니다.
![화면 캡처 2022-07-19 142121](https://user-images.githubusercontent.com/105689042/179682929-bd14e747-1994-4c1b-9c2c-a4ce4792175f.png)

혹시 에러 검출이 안 되는건 저만 그런건가요? 설정을 따로 해줘야 하는지 궁금합니다

**Comment by Cha-SeongHun** [comment]
저도 동일한 증상이 있습니다. 직접 실행해서 Exception이 발생해야만 오류가 있는지 알 수 있네요
어서 고쳐졌으면 좋겠습니다

**Reply by koldshine** [reply]
문제수정 패치됐다고 합니다

**Reply by chlguswls32** [reply]
visual Studio에서 자신이 TypeScript버전 몇을 쓰는지 어떻게확인할수있나요? 
위에 올려주신 스크린샷처럼 보는 방법을 못찾겠네요.

유니티에서는 첨부하는 이미지처럼 보고있는데, 비쥬얼스튜디오 코드에서 보는 방법도 궁금해서 질문드립니다.
![image](https://user-images.githubusercontent.com/39916808/181394100-26739c17-d635-41c8-b82d-1bbbd4a306f2.png)

**Reply by chlguswls32** [reply]
제페토 공식 팁 내용 위치도 질문드립니다.

**Reply by koldshine** [reply]
제가 참고한 글링크입니다. 
https://github.com/naverz/zepeto-studio-kor/discussions/481

저렇게 하셔도 되고요, 혹은 vscode 오른쪽 하단에 typescript괄호위치에 마우스 올려보시면 버전선택 팝업창이 작게 뜹니다
![화면 캡처 2022-07-28 093202](https://user-images.githubusercontent.com/105689042/181395290-a1f5e954-def0-4aca-9d68-549add85f55d.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 이슈 확인해 보겠습니다.

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 위 내용을 개선한 zepeto.script@1.4.2 패키지가 배포되어 안내 드립니다.

[개선 사항]
* 스크립트 에러 검출 오류 수정

[적용 방법]
Window → Package Manager 버튼을 눌러서 Package Manager를 오픈 → [ZEPETO.Script]를 누르고 오른쪽 하단의 Update 버튼을 누릅니다.
1.4.2 버전 업 후 컴파일러 재실행을 위해 유니티를 재시작 해 주세요.

**Reply by koldshine** [reply]
잘 되네요. 감사합니다
